src/SFA.DAS.RoATPService.Database/Tables/Audit.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[Audit]
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY (1,1), 
+    [OrganisationId] UNIQUEIDENTIFIER NOT NULL, 
```

Comment:
Would the org code that starts 'EPAxxxx' be better here?   They are unique, and the whole organisationId is kind of a left-over from starting days (in terms of the assessor-service, anyway).    I'll leave that with you, but you may want to discuss with Alan

Reply:
Ah scratch that, I just realised your organisations table doesn't even carry that field

Reply:
We don't use the EPAxxxx style org codes in our schema, the data that we're importing from the existing register has the UKPRN as the identifier